user-guide/modules/ROOT/pages/boost-history.adoc

```diff
@@ -25,28 +25,56 @@ One of the defining characteristics of Boost is its strict peer review process.
 
-* boost:spirit[] : A powerful and flexible parser and generator framework, which uses expression templates and template metaprogramming to create parsers at compile-time. This results in efficient and type-safe parsing code. Added to Boost *February 2003*, though has gone through multiple updates.
+* boost:graph[] : An extensive library for graph data structures and algorithms, enabling developers to work with graphs and network structures, while providing efficient implementations of popular graph algorithms like Dijkstra's, Kruskal's, and more.  Developed by Jeremy Siek, Lie-Quan Lee and Andrew Lumsdaine of Indiana University and released in *September 2000*.
 
-* boost:asio[] : A cross-platform library for network and low-level I/O programming, offering a consistent asynchronous model that allows developers to write efficient and highly-scalable applications. Added to Boost *April 2008*.
+* boost:spirit[] : A powerful and flexible parser and generator framework, which uses expression templates and template metaprogramming to create parsers at compile-time. This results in efficient and type-safe parsing code. Developed by Joel de Guzman and Hartmut Kaiser and added to Boost *March 2003*. 
 
-* boost:geometry[] : A library providing a collection of algorithms and data structures for working with geometrical objects, such as points, lines, polygons, and more. It also supports spatial indexing, coordinate system transformations, and various distance calculations. Added to Boost *July 2011*.
+* boost:asio[] : A cross-platform library for network and low-level I/O programming, offering a consistent asynchronous model that allows developers to write efficient and highly-scalable applications. Developed by Christopher Kohlhoff and added to Boost *March 2008*.
 
-* boost:compute[]: A library for parallel and high-performance computing on GPUs and other OpenCL devices, offering a high-level, easy-to-use interface for working with OpenCL. Added to Boost *November 2014*.
+* boost:geometry[] : A library providing a collection of algorithms and data structures for working with geometrical objects, such as points, lines, polygons, and more. It also supports spatial indexing, coordinate system transformations, and various distance calculations. Developed by Barend Gehrels, Bruno Lalande, Mateusz Loskot, Adam Wulkiewicz, Menelaos Karavelas and Vissarion Fisikopoulos and added to Boost *July 2011*.
 
-* boost:hana[] : A modern metaprogramming library for pass:[C++] that employs cutting-edge pass:[C++] features to facilitate powerful and expressive metaprogramming techniques, such as heterogeneous containers, compile-time algorithms, and type introspection. Added to Boost *May 2016*.
+* boost:algorithm[]: A library offering a collection of general purpose algorithm. Developed by Marshall Clow and added to Boost *June 2012*.
```

Comment:
So has this one.

Reply:
OK, made a note of this for both libraries.

Reply:
Boost heap (priority queues) were incorporated in the standard library https://en.cppreference.com/w/cpp/container/priority_queue